src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -87,6 +89,67 @@ describe('<ResourceItem />', () => {
     });
   });
 
+  describe('name', () => {
+    it('is used on the Checkbox label when item is not selected', () => {
+      const item = mountWithAppProvider(
+        <ResourceListContext.Provider value={mockSelectableContext}>
+          <ResourceItem
+            accessibilityLabel={accessibilityLabel}
+            id={itemId}
+            url="https://shopify.com"
+            name={name}
+          />
+        </ResourceListContext.Provider>,
+      );
+
+      const expectedLabel = `Select: ${name}`;
```

Comment:
Checkboxes report their selected or unselected state by default, so we should avoid including that language in the label. This also helps with translation. Allowing the label to just be `${name}` should provide the support we need. (This is something I've mentioned we should change for the resource list in #792, as well.)

Basically, I'd expect a unique `name` value for each checkbox. If the consumer wants to override the default, they could provide their own in the `accessibilityLabel`. Does that make sense?

Reply:
I've made the change. I left the old `accessibility label` as a fallback, so that there are no unexpected changes for people who had defined it but don't have name yet.